src/main/resources/assets/cubicgen/lang/en_us.lang

```diff
@@ -42,21 +42,30 @@ cubicgen.gui.cubicgen.from=From:
 ###   basic terrain settings  ###
 #################################
 
+cubicgen.gui.cubicgen.biome=Biome
+cubicgen.gui.cubicgen.biomeSize=Biome Size
+cubicgen.gui.cubicgen.riverSize=River Size
+
 cubicgen.gui.cubicgen.water_level=Ocean Level
+cubicgen.gui.cubicgen.lavaOceans=Lava Oceans
+
 cubicgen.gui.cubicgen.caves=Caves
-cubicgen.gui.cubicgen.strongholds=Strongholds
-cubicgen.gui.cubicgen.villages=Villages
-cubicgen.gui.cubicgen.mineshafts=Mineshafts
-cubicgen.gui.cubicgen.temples=Temples
-cubicgen.gui.cubicgen.oceanMonuments=Ocean Monuments
-cubicgen.gui.cubicgen.woodlandMansions=Woodland Mansions
 cubicgen.gui.cubicgen.ravines=Ravines
+
+cubicgen.gui.cubicgen.mineshafts=Mineshafts*
+
 cubicgen.gui.cubicgen.dungeons=Dungeons
 cubicgen.gui.cubicgen.dungeonCount=Dungeon Count
-cubicgen.gui.cubicgen.lavaOceans=Lava Oceans
-cubicgen.gui.cubicgen.biome=Biome
-cubicgen.gui.cubicgen.biomeSize=Biome Size
-cubicgen.gui.cubicgen.riverSize=River Size
+
+cubicgen.gui.cubicgen.strongholds=Strongholds
+
+cubicgen.gui.cubicgen.villages=Villages*
+
+cubicgen.gui.cubicgen.temples=Temples*
+
+cubicgen.gui.cubicgen.oceanMonuments=Ocean Monuments*
+
+cubicgen.gui.cubicgen.woodlandMansions=Woodland Mansions*
```

Comment:
Why is it moved?

Reply:
to have structures grouped together, should this be reverted?